src/celestia/qt/qtmain.cpp

```diff
@@ -16,33 +16,34 @@
  ***************************************************************************/
 
 
-#include <QTime>
+#include <cstdio>
+#include <iostream>
+#include <string>
+
+#include <fmt/printf.h>
+
 #include <QApplication>
-#include <QSplashScreen>
+#include <QBitmap>
 #include <QDesktopServices>
 #include <QDir>
+#include <QLibraryInfo>
+#include <QLocale>
 #include <QPixmap>
-#include <QBitmap>
+#include <QSplashScreen>
+#include <QTime>
+
 #include "qtgettext.h"
-#include <QLocale>
-#include <QLibraryInfo>
-#include <vector>
 #include "qtappwin.h"
-#include <fmt/printf.h>
-
-using namespace std;
-
-//static const char *description = "Celestia";
 
 // Command line options
 static bool startFullscreen = false;
 static bool runOnce = false;
-static QString startURL;
-static QString logFilename;
-static QString startDirectory;
-static QString startScript;
-static QStringList extrasDirectories;
-static QString configFileName;
+static QString startURL{ };
```

Comment:
why to add these {}? static variables are initialized by default.

Reply:
because honestly I can't remember the rules, so I assume anything not initialized is probably undefined behavior.

Reply:
And furthermore, what we had before is technically zero-initializing the QString, which depending on the internals of the Qt framework may or may not be the same as default-initializing a QString. I prefer to err on the side of caution.

Reply:
I guess we might also ask why bother initializing the static booleans to false when they will be zero-initialized?